calico_kubernetes/calico_kubernetes.py

```diff
@@ -591,20 +614,37 @@ def _label_to_tag(self, label_key, label_value):
         tag = self._escape_chars(tag)
         return tag
 
+
+def config_logger():
```

Comment:
I might actually move this to a new file, "logging.py" or something, to keep it separated from plugin code.